ndbench-core/src/main/java/com/netflix/ndbench/core/config/IConfiguration.java

```diff
@@ -89,7 +96,25 @@ default int getNumReaders() {
     @DefaultValue("100")
     int getReadRateLimit();
 
+
     @DefaultValue("100")
+    @PropertyName(name=WRITE_RATE_LIMIT)
     int getWriteRateLimit();
 
+
+    @DefaultValue("false")
+    boolean isAutoTuneEnabled();
+
+
+    @DefaultValue("60")
```

Comment:
the main usage is in the project nfndbench.  but vinay asked me to write unit test wi/ stubs in this project as well.. which i will partially complete tonight.